package.json

```diff
@@ -213,7 +213,9 @@
         "yargs": "^16.2.0"
     },
     "resolutions": {
-        "cssnano/**/postcss-calc": "7.0.0"
+        "cssnano/**/postcss-calc": "7.0.0",
+        "lit-element": "3.0.0-rc.2",
```

Comment:
@Westbrook since you removed the lit-element and lit-html dependencies above, is this still required?